hera_cal/vis_clean.py

```diff
@@ -43,13 +45,21 @@ def __init__(self, input_data, filetype='uvh5', input_cal=None, link_data=True,
                 as they are built.
             link_data : bool, if True, attempt to link DataContainers
                 from HERAData object, otherwise only link metadata if possible.
+            spw_range : range of frequencies to delay-filter / write out.
```

Comment:
clarify the format and datatype expected here. Also clarify that it's channels and not frequencies. Also, why are we doing channels? hera_cal usually frequencies wherever possible

Reply:
Also, what's the spw_range for? When is it used? Doesn't it make more spense to attach to a specific filter function or read function (it's already basically supported in the read functions). 

I'd remove this...

Reply:
`spw_range` allows for the user to create a `vis_clean` object that only filters across a subband and saves data files that only include said subband. 

it is also used to select proper subbands for any attached `HERACal` objects. 

The alternative, that I think you are suggesting, is that one could initialize a vis_clean object with `link_data=False` and then run `read(freqs = frequencies_of_interest)` 

If we did things this way, there would be several extra steps for the user (rather then just initializing with an spw_range). They would have to call `read(freqs=frequencies_of_interest)` afterwards and we would have to keep the extra code in `apply_calibration` but now change from `spw_range` to the attached data frequencies. 

So I guess the tradeoff would be less code for different (depending on who you are less clear) usability. Since the rest of `io` uses `read` as a common way of downselecting data, I'm inclined to go with the `read` method instead of the `spw_range` method.